native-auth/config.md

```diff
@@ -9,3 +9,41 @@ dto NativeAuthClientConfig:
     gateway_url?: string;
     extra_request_headers?: dict[string, Any];
 ```
+
+## NativeAuthServerConfig
+
+```
+dto NativeAuthServerConfig:
+    // The endpoint from where the current block information will be fetched upon validation.
+    api_url: string;
+
+    // A mandatory list of accepted origins in case the server component must validate the incoming requests by domain.
+    accepted_origins: string[];
+
+    // The endpoint where the impersonation is validated
+    // This is called if the extraInfo payload contains the `multisig` or `impersonate` attribute.
+    validate_impersonate_url?: string;
+
+    // An optional function that returns a boolean if the impersonation is accepted
+    // This is called if the extraInfo payload contains the `multisig` or `impersonate` attribute.
+    validate_impersonate_callback?: string;
```

Comment:
Implementation must be handled with care (impersonation on dApp back-ends can be a sensitive topic).